README.md

```diff
@@ -44,30 +44,25 @@ different networks, you can read more about this in [this article](https://tezos
 
 The following distributions are supported by the `tezos-packaging`:
 
-<a name="ubuntu"></a>
 ### [Ubuntu](./docs/distros/ubuntu.md)
 
 Native Ubuntu packages that can be installed with `apt-get` from a PPA.
 
-<a name="raspberry"></a>
 ### [Raspberry Pi OS](./docs/distros/ubuntu.md#raspberry)
 
 Some Raspverry systems also support packages that can be installed using `apt-get`.
```

Comment:
```suggestion
Some Raspberry systems also support packages that can be installed using `apt-get`.
```
Uh, there's also a typo here that was missed.

Reply:
:facepalm: